openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/TotalEntityInsight.tsx

```diff
@@ -129,8 +130,8 @@ const TotalEntityInsight: FC<Props> = ({ chartFilter, selectedDays }) => {
         </>
       }>
       {data.length ? (
-        <Row gutter={16}>
-          <Col span={18}>
+        <Row gutter={DI_STRUCTURE.rowContainerGutter}>
```

Comment:
I can recommend another way to do so let's connect for the same.
Other then approach rest looks good to me